app/src/main/java/com/infomaniak/mail/ui/main/menuDrawer/MenuDrawerAdapter.kt

```diff
@@ -243,59 +246,59 @@ class MenuDrawerAdapter @Inject constructor() : ListAdapter<Any, MenuDrawerViewH
     abstract class MenuDrawerViewHolder(val binding: ViewBinding) : ViewHolder(binding.root)
 
     private enum class ItemType {
-        CUSTOM_FOLDERS_HEADER,
-        EMPTY_CUSTOM_FOLDERS,
-        DIVIDER,
+        DIVIDER_ITEM,
+        FOLDERS_HEADER_ITEM,
+        EMPTY_CUSTOM_FOLDERS_ITEM,
     }
 
-    private enum class NotifyType {
-        MAILBOXES_HEADER_CLICKED,
+    enum class ViewType {
+        DIVIDER_VIEW,
+        MAILBOXES_HEADER_VIEW,
+        MAILBOX_VIEW,
+        INVALID_MAILBOX_VIEW,
+        FOLDERS_HEADER_VIEW,
+        FOLDER_VIEW,
+        EMPTY_FOLDERS_VIEW,
+        FOOTER_VIEW,
```

Comment:
Did you try making ItemType and ViewType a single enum?

Reply:
Done.